setup.cfg

```diff
@@ -11,48 +9,50 @@ description = Community-developed python astronomy tools
 long_description = file: README.rst
 keywords = astronomy, astrophysics, cosmology, space, science,units, table, wcs, samp, coordinate, fits, modeling, models, fitting, ascii
 classifiers =
-    Intended Audience :: Science/Research
-    License :: OSI Approved :: BSD License
-    Operating System :: OS Independent
-    Programming Language :: C
-    Programming Language :: Cython
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: Implementation :: CPython
-    Topic :: Scientific/Engineering :: Astronomy
-    Topic :: Scientific/Engineering :: Physics
+	Intended Audience :: Science/Research
+	License :: OSI Approved :: BSD License
+	Operating System :: OS Independent
+	Programming Language :: C
+	Programming Language :: Cython
+	Programming Language :: Python :: 3
+	Programming Language :: Python :: Implementation :: CPython
+	Topic :: Scientific/Engineering :: Astronomy
+	Topic :: Scientific/Engineering :: Physics
 
 [options]
+packages = find:
```

Comment:
What is this? Maybe add comment? People will look to this `setup.cfg` for inspiration in affiliated packages... (well, I do)

Reply:
This will cause all folders containing an ``__init__.py`` to be found. I'll add a comment.

Reply:
Done